build.gradle

```diff
@@ -38,6 +38,7 @@ subprojects { project ->
         errorproneJavac deps.build.errorProneJavac
     }
     project.tasks.withType(JavaCompile) {
+        dependsOn(installGitHooks)
```

Comment:
out of curiosity, does this task cache correctly? Or just cheap enough to run every time?

Reply:
Actually I see here: https://docs.gradle.org/current/userguide/build_cache.html#sec:task_output_caching_details

> Some tasks, like Copy or Jar, usually do not make sense to make cacheable because Gradle is only copying files from one location to another. It also doesn’t make sense to make tasks cacheable that do not produce outputs or have no task actions.

I wonder if there's a friendlier way to handle this, such as using the spotless gradle plugin

Reply:
This won't be cached, but involves only copying a single 358 bytes file, so checking the cache would not be any faster, I think :) (and makes sure the hook gets re-added if the user removes it somehow). Is there a problem with it I am not seeing?